### DISCUSSION TITLE
Frontend broken?

### DISCUSSION THREAD

**Discussion by openstepmedia** [SonicJs-Org/sonicjs#78] [Q&A] [answered]
Hi. I installed sonicjs without any issues, created my admin account, all good.  

The frontend of the site seems to be broken (master branch). At first I noticed some js errors in chrome tools, fancybox wasn't installed, so I installed that and the console errors went away. 

Issue now is that the Column / Module / Row buttons are not responding.

See below.. the console has some debug code there, but I'm not sure what's going on with the buttons to add / manipulate content.

Thanks for your help!

![image](https://user-images.githubusercontent.com/876400/129419091-0e70c112-d755-4497-b8e0-696ec62795fd.png)

[edit]
I think the issue is here:

https://github.com/lane711/sonicjs/blob/master/server/page-builder/js/page-builder.js#L193

I'm guessing that the click on the content row is swallowing the click for the btnGroupDropModules dropdown-menu.

**Comment by lane711** [comment]
Hi @openstepmedia, thanks for submitting this. I am unable to replicate this is chrome. What browser are you using?

**Comment by lane711** [comment, marked as answer]
This issue should be resolved in the latest master. thanks!
